src/contexts/useArchaeologistService.ts

```diff
@@ -0,0 +1,93 @@
+import { useState } from 'react';
+import Arweave from 'arweave';
+
+const initArweave = () => {
+  return Arweave.init({
+    host: 'localhost', // 'arweave.net', // Hostname or IP address for a Arweave host
```

Comment:
Kyle and I discussed having a config switch for the arlocal settings. So in production it would never use localhost. but you could choose which one in env.dev. There is a note on the Handle Multiple Networks card